server/service/src/processors/mod.rs

```diff
@@ -64,7 +64,11 @@ impl Processors {
 
         tokio::spawn(async move {
             loop {
+                // See test below for reasoning behind biased, even though there is no foreseen use case wehere
```

Comment:
I'm wondering how the comment relates to #613 which appears to outline a use case where the order is important

Reply:
@mark-prins I first did the biased fix, but then realised that it could be fixed with more resilient logic in processors rather then the driver. Ended up making PR for both (and I changed the comment here to say there is no foreseen use cases, rather then that specific requisition link use case). Potentially there are use cases where this is important, but I can't think of any more atm, and having this logic makes it easier to think through the logic